### DISCUSSION TITLE
Please Provide binaries for x64 pc

### DISCUSSION THREAD

**Discussion by SonaliBendre** [zhongfly/mpv-build#5] [Q&A] [answered]
Please Provide binaries for x64 pc

**Comment by zhongfly** [comment, marked as answer]
The release is for x64,work fine in my x64 pc

**Reply by zhongfly** [reply]
same as https://sourceforge.net/projects/mpv-player-windows/files/ "x86_64" means build for 64 bit pc

**Comment by SonaliBendre** [comment]
Oh sorry

**Reply by zhongfly** [reply]
Thanks for your questions.This let me know that my work is useful.

**Reply by SonaliBendre** [reply]
Yes sir you read my mind and made this
I mean I was searching Github Actions method for automatic build for many days as sourceforge sucks

**Comment by SonaliBendre** [comment]
BTW can you write Github Action for mpv.net & mpc-qt?

**Reply by zhongfly** [reply]
Sorry, I don't know how to build those projects.

**Reply by SonaliBendre** [reply]
mpv.net & mpc-qt are just frontend for mpv, there's instruction in the project readme.MD/doc how to compile it for windows
https://github.com/mpc-qt/mpc-qt
https://github.com/stax76/mpv.net
I understand that you never used these and familiar with these, I'll test if I my Actions works

**Reply by zhongfly** [reply]
> mpv.net & mpc-qt are just frontend for mpv, there's instruction in the project readme.MD/doc how to compile it for windows https://github.com/mpc-qt/mpc-qt https://github.com/stax76/mpv.net I understand that you never used these and familiar with these, I'll test if I my Actions works

about mpv.net you can copy this https://github.com/dyphire/mpv.net/blob/master/.github/workflows/mpvnet-build.yml

**Comment by SonaliBendre** [comment]
@zhongfly Are you on Telegram? Can we talk there?

**Reply by zhongfly** [reply]
I am very sorry, I am not a professional programmer, and I cannot provide you with any help with those projects. I suggest that you can try to build, and directly ask the developers of these projects for help when you encounter problems, I believe they will help you

**Reply by SonaliBendre** [reply]
Ok Thanks, I will try to build using Github Actions (I'm very lazy though)

**Comment by SonaliBendre** [comment]
BTW what is mpv-dev build?

**Reply by zhongfly** [reply]
`mpv-1.dll`
 player based on libmpv need this,such as mpv.net

**Reply by SonaliBendre** [reply]
Thanks
